docs/services/services.md

```diff
@@ -0,0 +1,10 @@
+# Siembol's services:
```

Comment:
'Siembol services' to be consistent?

Reply:
done!

Reply:
Perfect! Can you please remove ':' ? Then we can merge it.

Reply:
done!